### PR TITLE
Added Egress Iris solution

### PR DESCRIPTION
**Pull request by tonytilbo**
Adds a new Azure Sentinel Solution for the Egress Iris Data platform, currently has a single data connector and workbook, we will be adding more content in future.

### CODE REVIEW

**Comment by tonytilbo** [comment]
@microsoft-github-policy-service agree [company="Egress Software Technologies"]

**Comment by tonytilbo** [comment]
@microsoft-github-policy-service agree company="Egress Software Technologies"

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, Thanks for raising this PR. 
This PR will be investigated and we will update you about the same before 01 April, 2024.

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, Can you proceed with the above requested changes

**Comment by tonytilbo** [comment]
@v-prasadboke apologies, it was a long weekend in the UK so will look into this today.

**Comment by v-prasadboke** [comment]
Thanks for the updates @tonytilbo

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, Workbookmetadata is missing can you please add it in https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, We are waiting for your updates

**Comment by tonytilbo** [comment]
That has been added now.

**Comment by v-prasadboke** [comment]
And looks like you have deleted some of the content from other Solution. Any prior reason

File: .vscode/settings.json [hunk 1]
```diff
@@ -14,5 +14,6 @@
     },
     "githubPullRequests.ignoredPullRequestBranches": [
         "master"
-    ]
+    ],
+    "dotnet.defaultSolution": "disable"
```
Comment (v-prasadboke) [inline .vscode/settings.json:18]: Any reason behind changing this.

**Comment by v-prasadboke** [comment]
Also please add working images of workbook with black and white background. Also add Solution's logo to logo folder

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, Please work on the changes mentioned above

File: .vscode/settings.json (hunk 1 above)
Comment (tonytilbo) [reply .vscode/settings.json:18]: Not sure how that got in but change has been reverted.

**Comment by v-prasadboke** [comment]
Thanks for committing the required changes

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, Sample data is incomplete 
Please provide complete sample data to test the workbook

**Comment by tonytilbo** [comment]
> Hello @tonytilbo, Sample data is incomplete Please provide complete sample data to test the workbook

@v-prasadboke is there some example of how to provide sample data and I'll get that sorted.

**Comment by v-prasadboke** [comment]
Hello @tonytilbo, 
Please provide your response on the above comment

**Comment by tonytilbo** [comment]
@v-prasadboke we are just pulling some sample data together, once I have this I'll get it included.

**Comment by tonytilbo** [comment]
@v-prasadboke sample data has been added.

**Review by v-prasadboke — APPROVED** [review]

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**